.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,141 @@
+name: Déploiement
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches: [master, demo, next]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    outputs: 
+      fr: ${{ env.FR_SITE }}
+      en: ${{ env.EN_SITE }}
+      publicodes: ${{ env.PUBLICODES_SITE }}
+    steps:
+      - name: Set branch name
+        id: vars
+        run: echo ::set-output name=branch::${GITHUB_REF#refs/*/}
+      - if: github.event_name == 'pull_request'
+        name: 'Set site name for pull request deploy'
+        run:
+          echo "FR_SITE=https://deploy-preview-${{ github.event.number }}--syso.netlify.app/\${path}" >> $GITHUB_ENV;
```

Comment:
Salut ! On doit pouvoir utiliser la commande [`env`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env) ?

Reply:
Salut :wave:

Mmmh, cela me paraissait plus simple à faire via des étapes conditionnelles que d'ajouter une expression à rallonge dans env pour changer le nom en fonction du github event...

Reply:
On peut utiliser le `env` dans l'étape conditionnelle : https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsenv

Reply:
Oui mais du coup, les variables ne s'applique qu'au step courant (ce qui est logique). Enfin, c'est ce que j'en ai compris...